src/Sdk/HistoryApi.php

```diff
@@ -32,6 +36,89 @@ public function listOperations(
         return $this->dcoreApi->requestWebsocket(History::class, new GetAccountHistory($accountId, $startId, $endId, $limit)) ?: [];
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function listTransferOperationsComposed(
```

Comment:
Please rename to findAllTransfersComposed to align with JS SDK

Reply:
Renamed to `findAllTransfersComposed `